### ISSUE TITLE
Using module standalone

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/vcf-js#6]
If I try to require('@gmod/vcf') in a standalone file I get an error

```
node server.js
internal/modules/cjs/loader.js:615
    throw err;
    ^

Error: Cannot find module 'babel-runtime/core-js/object/assign'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:613:15)
    at Function.Module._load (internal/modules/cjs/loader.js:539:25)
    at Module.require (internal/modules/cjs/loader.js:667:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/cdiesh/src/protein_data_service/node_modules/@gmod/vcf/dist/parse.js:1:90)
    at Module._compile (internal/modules/cjs/loader.js:738:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
    at Module.load (internal/modules/cjs/loader.js:630:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
    at Function.Module._load (internal/modules/cjs/loader.js:562:3)
```

Might be sort of similar to this https://github.com/GMOD/twobit-js/issues/1

**Comment by cmdcolin** [comment]
Manually installing babel-runtime is a workaround but just curious if that should be required to use module or otherwise documented

**Comment by garrettjstevens** [comment]
Looks like babel-runtime is [supposed to be a dependency](https://babeljs.io/docs/en/babel-plugin-transform-runtime), but it's not in the `package.json`. tabix-js has the same problem, too. This is a good excuse to upgrade both to babel 7, I'll work on that.

**Comment by cmdcolin** [comment]
Thanks, good catch on it being a regular dependency...I was having a hard time interpreting that section

**Comment by garrettjstevens** [comment]
This fix is in v2.0.3, just published to NPM
